src/pages/trails/ballots/components/BallotForm.vue

```diff
@@ -708,100 +771,103 @@ q-dialog(
-              .q-mt-sm
-                b DAO:&nbsp;
-                span {{form.treasurySymbol.label}}
-              .q-mt-sm
-                .flex.row.gap
-                  .col.flex.column
-                    b options:
-                    ul.q-mt-xs
-                      li(v-for="(option,index) in form.optionsLabels") {{option}}
-                  .col.flex.column
-                    br
-                    span(v-if="onlyOneOption") Voters only can choose one option
-                    span(v-if="!onlyOneOption") Voters can choose from {{form.minOptions}} to {{form.maxOptions}} options
-              .q-mt-sm
-                b open:&nbsp;
-                span(v-if="openForVoting") this ballot will be open from now until {{moment(form.endTime).format("MMMM Do YYYY")}}
-                span(v-if="!openForVoting") this ballot will not open immediately and must be oppened manually in the future
+          q-card-section(:class="smallDevice ? 'q-px-none q-py-xs' : ''")
+            .q-mt-sm.flex.gap(:class="previewVertical ? 'column' : 'raw'")
+              .col-auto.flex.column.preview-left.row-direction(v-if="!previewVertical")
+                ballot-list-item(
+                  :ballot="ballot"
+                  :displayWinner="false"
+                  :isBallotOpened="false"
+                  :startTimeHasPassed="false"
+                  :getStartTime="getStartTime()"
+                  :getEndTime="form.endTime"
+                  :getLoser="false"
+                )
+              .col.flex.column.preview-right
+                .q-mt-sm
+                  b title:
+                  br
+                  span {{form.title}}
+                .q-mt-sm
+                  b description:
+                  br
+                  span {{form.description}}
+                .q-mt-sm
+                  b PDF:&nbsp;
+                  span(v-if="form.file") {{filename}}
+                  span(v-if="!form.file") No pdf was attached
+                .q-mt-sm
+                  b DAO:&nbsp;
+                  span {{form.treasurySymbol.label}}
+                .q-mt-sm
+                  .flex.row.gap
+                    .col.flex.column
+                      b options:
+                      ul.q-mt-xs
+                        li(v-for="(option,index) in form.optionsLabels") {{option}}
+                    .col.flex.column
+                      b method:
+                      span(v-if="onlyOneOption") Voters only can choose one option
+                      span(v-if="!onlyOneOption") Voters can choose from {{form.minOptions}} to {{form.maxOptions}} options
+                .q-mt-sm
+                  b open:&nbsp;
+                  span(v-if="openForVoting") this ballot will be open from now until {{moment(form.endTime).format("MMMM Do YYYY")}}
+                  span(v-if="!openForVoting") this ballot will not open immediately and must be oppened manually in the future
```

Comment:
you can just use `v-else` here

Reply:
fixed here [d59e224](https://github.com/telosnetwork/app-telos-native/pull/223/commits/d59e22409bbdd61d722885cee0ec899727e2da68)